README.md

```diff
@@ -63,66 +62,66 @@ This script sets up some boilerplate dotfiles and installs common tools.
 
 The following things are set up:
 
-  * Generates a new SSH key pair if you don't have one yet
-  * Copies the following dotfiles to your homefolder
-  * Creates a `~/Projects` folder
-  * Installs Brew and common Brew packages
-  * Installs OH-MY-ZSH
-  * Installs some common global Node.js packages
-  * Installs the Phoenix Framework
-  * Installs the Vim plugins configured in the `.vimrc` file
-  * Changes your default shell to ZSH
+* Generates a new SSH key pair if you don't have one yet
+* Installs Brew and common Brew packages
+* Installs Node using [Volta](https://volta.sh/)
```

Comment:
I'm not familiar with Volta. Is there a reason why we're using Volta for Node and not asdf? Thanks!

Reply:
Volta is Node/JS specific, so if you're on that kind of project, then it's great.  asdf covers more languages (like Exlixr, Node, etc.).  Any reason to not install both?

Reply:
I'm new to Volta and not totally sure but it seems an improvement over nvm and I have not had a good time personally using asdf for Node/npm. Volta and nvm at least will switch to a specified version when you change directory, and Volta does this more intelligently than nvm.

Reply:
Ah thanks for the info about Volta. `nvm` was a good solution at one time, but gosh it became cumbersome to deal with compared to `asdf`. That being said I can see Volta being easier to deal with `asdf`